Diffusion/Common/State.swift

```diff
@@ -71,14 +72,14 @@ class GenerationContext: ObservableObject {
 
     func generate() async throws -> GenerationResult {
         guard let pipeline = pipeline else { throw "No pipeline" }
-        let seed = self.seed >= 0 ? UInt32(self.seed) : nil
+        let seed = self.seed > 0 ? UInt32(self.seed) : nil
```

Comment:
Good catch. The change here is to ensure it's greater than 0 (the new random indicator) and not equal to 0 (as unsigned int cannot be negative.) Since the cast to UInt32 was already there I failed to remove it! I shall include the removal of the cast in the next commit as well.